src/clothing_mod.h

```diff
@@ -12,10 +12,11 @@
 #include "calendar.h"
 #include "bodypart.h"
 
-class JsonObject;
 class JsonArray;
-class player;
+class JsonObject;
 class item;
+class player;
```

Comment:
This was unused so it was removed, no need to add it back.

Reply:
My bad, I'll get it fixed.